app/helpers/application_helper.rb

```diff
@@ -60,6 +60,23 @@ def user_options
     users.all.collect { |u| [u.user_key, u.display_name] }
   end
 
+  def status_span_generator(status)
+    case status
+    when 'submitted'
+      tag(:span, status.titleize, class: 'job-status badge rounded-pill bg-default')
+    when 'errored'
+      tag(:span, status.titleize, class: 'job-status badge rounded-pill bg-danger')
+    when 'completed'
+      tag(:span, status.titleize, class: 'job-status badge rounded-pill bg-success')
+    when 'in_progress'
+      tag(:span, status.titleize, class: 'job-status badge rounded-pill bg-warning')
+    when 'Unknown'
+      tag(:span, status.titleize, class: 'job-status badge rounded-pill')
+    else
+      tag(:span, '📖 Banana', class: 'job-status badge rounded-pill bg-info')
```

Comment:
This syntax is deprecated and it appears to throw errors in the test suite

use `tag.span(...` instead of `tag(:span ...`